src/utility/svd.jl

```diff
@@ -438,13 +438,21 @@ function ChainRulesCore.rrule(
     return (U, S, V, info), tsvd!_itersvd_pullback
 end
 
-# scalar inverses with a cutoff tolerance and Lorentzian broadening
-function _safe_inv(x, tol, ε=0)
-    if abs(x) < tol
-        return zero(x)
-    else
-        return iszero(ε) ? inv(x) : _lorentz_broaden(x, ε)
+# scalar inverses with a cutoff tolerance
+_safe_inv(x, tol) = abs(x) < tol ? zero(x) : inv(x)
+
+# compute inverse singular value difference contribution to SVD gradient with broadening ε
+function _broadened_inv_S(S::AbstractVector{T}, tol, ε=0) where {T}
+    F = Matrix{T}(undef, length(S), length(S))
+    @inbounds for i in axes(F, 1), j in axes(F, 2)
```

Comment:
```suggestion
    @inbounds for j in axes(F, 2), i in axes(F, 1)
```

Inner loops should be columns 😉

Reply:
Whoopsie, rookie mistake! :)